.github/workflows/fuzzing-run.yml

```diff
@@ -0,0 +1,61 @@
+name: Fuzzer execution
+
+on:
+  workflow_call:
+    inputs:
+      target:
+        required: true
+        type: string
+      arch:
+        required: true
+        type: string
+      package-type:
+        required: true
+        type: string
+      timeout-seconds:
+        description: The maximum time in seconds to run the fuzzer.
+        required: false
+        type: number
+        default: 600
+
+jobs:
+  package:
+    uses: ./.github/workflows/package-build.yml
+    with:
+      target: ${{ inputs.target }}
+      arch: ${{ inputs.arch }}
+      artifact: ${{ inputs.target }}-fuzzer
+      package-type: ${{ inputs.package-type }}
+      build-fuzzers: true
+
+  fuzzer-execution:
+    needs: package
+    runs-on: [self-hosted, 1ES.Pool=e2e-pool, '1ES.ImageOverride=${{ inputs.target }}']
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/download-artifact@v4
+        id: download
+        with:
+          name: ${{ inputs.target }}-fuzzer
+
+      - name: Install prerequisites
+        run: |
+          sudo apt update
+          sudo apt install -y clang
```

Comment:
this won't be needed once the container changes are in

Reply:
That's needed at runtime. I think the VM image is different than the container image we use for building. If I'm wrong, I'll drop it later when the prerequisite PR is merged.

Reply:
For sure the llvm-symbolizer is needed at runtime, that's part of the llvm package which is a dependency of clang.

Reply:
ahh yes indeed, good point! We should also add this into our managed image!

Reply:
is that controlled by `devops/e2e/cloudtest/` or in some other place?